packages/interpretations/src/components/Lists/CommentsList.js

```diff
@@ -58,7 +60,9 @@ export class CommentsList extends React.Component {
         comment.save(this.context.d2).then(() => this.props.onChange(this.props.interpretation));
     };
 
-    onDeleteComment(comment) {
+    onDeleteComment() {
+        const comment = this.state.commentToDelete;
+        console.log('comment to delete', comment);
```

Comment:
I guess this was left over from testing?

Reply:
If so, the `const` isn't needed since it's only used once.

Reply:
yep, i'm going to clean it up.